CHANGES.md

```diff
@@ -7,6 +7,7 @@ Release Notes.
 #### Project
 * Incompatible with previous releases when use H2/MySQL/TiDB storage options, due to support multiple alarm rules triggered for one entity.
 * Chore: adapt `create_source_release.sh` to make it runnable on Linux.
+* Add `package` to `.proto` files, prevent polluting top-level namespace in some languages.
```

Comment:
Please highlight the agent would not work for previous releases.

Reply:
I just modified 2-3 LOC that avoid the breaking changes, please check whether that’s worthy or not

Reply:
This commit. https://github.com/apache/skywalking/pull/6026/commits/42ca4e880f2097bc9e175cc6ee651975ffb457db